src/uct/ib/rc/verbs/rc_verbs_iface.c

```diff
@@ -230,10 +229,9 @@ static ucs_status_t uct_rc_verbs_iface_query(uct_iface_h tl_iface, uct_iface_att
     iface_attr->latency.m += 1e-9;  /* 1 ns per each extra QP */
     iface_attr->overhead   = 75e-9; /* Software overhead */
 
-    iface_attr->ep_addr_len = sizeof(uct_rc_verbs_ep_address_t);
-    if (md->ops->get_atomic_mr_id(md, &mr_id) == UCS_OK) {
-        iface_attr->ep_addr_len += sizeof(mr_id);
-    }
+    iface_attr->ep_addr_len = uct_ib_md_is_flush_rkey(md) ?
+                              sizeof(uct_rc_verbs_ep_flush_addr_t) :
+                              sizeof(uct_rc_verbs_ep_addr_t);
```

Comment:
what if flush is not supported, but atomic is supported?

Reply:
should not be: there is same condition to support for flush remote & atomic (KSM)

Reply:
then better to expose this explicitly (maybe change naming)

Reply:
ok, reverted